Quadriga/src/main/java/edu/asu/spring/quadriga/web/resolver/TestResolverController.java

```diff
@@ -0,0 +1,43 @@
+package edu.asu.spring.quadriga.web.resolver;
+
+import java.security.Principal;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import edu.asu.spring.quadriga.domain.resolver.IProjectHandleResolver;
+import edu.asu.spring.quadriga.domain.resolver.impl.ProjectHandleResolver.STATUS;
+import edu.asu.spring.quadriga.service.resolver.IProjectHandleResolverManager;
+
+@Controller
+public class TestResolverController {
+
+    @Autowired
+    private IProjectHandleResolverManager resolverManager;
+
+    @RequestMapping(value = "/auth/resolvers/test", method = RequestMethod.POST)
+    public String editPage(Principal principal, Model model, @RequestParam("resolverId") String resolverId,
+            RedirectAttributes redirectAttributes) {
+
+        IProjectHandleResolver projectHandleResolver = resolverManager.getProjectHandleResolver(resolverId);
+        String resolvedHandle = projectHandleResolver.buildResolvedHandle(projectHandleResolver.getHandleExample());
+
+        if ((projectHandleResolver.getResolvedHandleExample()).equals(resolvedHandle)) {
```

Comment:
just add a method to the manager class that checks the regex pattern and sets the project handle resolver validation field if required.